eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -55,6 +55,21 @@ jobs:
         inputs:
           versionSpec: '3.6'
 
+      - template: tools/daily-dev-build-variable/daily-dev-build-variable.yml@azure-sdk-tools
+
+      - ${{ each artifact in parameters.Artifacts }}:
+        - script: |
+            python3 --version 
+            python3 eng/versioning/set_versions.py --update-type all --build-type client --build-qualifier dev.$(Build.BuildNumber) --artifact-id ${{artifact.name}}
+          condition: eq(variables['SetDevVersion'],'true')
+          displayName: Apped dev package version suffix for ${{artifact.name}}
```

Comment:
"Apped" -> "Append" .

Any reason to not do both of these steps in the same script block?

Reply:
The first script block is executed multiple times, once per artifact specified in ci.yml, the second block is executed once. Will fix the typo.